src/main/scala/mailo/http/SendinblueClient.scala

```
@@ -0,0 +1,155 @@
+import akka.http.scaladsl.model._
+import akka.http.scaladsl.model.headers.{Authorization, BasicHttpCredentials, RawHeader}
+import akka.http.scaladsl.marshalling._
+import akka.http.scaladsl.unmarshalling._
+
+import akka.stream.ActorMaterializer
+import akka.stream.scaladsl.Source
+import akka.actor.ActorSystem
+
+import scala.concurrent.{Future, ExecutionContext}
+import javax.mail.Message.RecipientType
+import javax.mail.internet.MimeMessage
+
+import com.typesafe.config.{Config, ConfigFactory}
+
+import cats.data.EitherT
+import cats.syntax.either._
+import cats.instances.future._
+import cats.instances.either._
+
+import akka.http.scaladsl.model.ContentType
+import akka.util.ByteString
+
+import scala.util.Try
+import util._
+import com.sendinblue.Sendinblue
+
+import io.circe._, io.circe.generic.auto._, io.circe.parser.decode
+
+class SendinblueClient(
+  implicit
+  system: ActorSystem,
+  materializer: ActorMaterializer,
+  conf: Config = ConfigFactory.load()
+) extends MailClient
+    with MimeMailClient
+    with LazyLogging {
+  import MailClientError._
+  import mailo.MailRefinedContent._
+  import mailo.MailResponse
+  import mailo.MailError
+
+  private[this] case class SendinblueConfig(key: String)
+  private[this] val sendinblueConfig = SendinblueConfig(
+    key = conf.getString("mailo.sendinblue.key")
+  )
+  private[this] val sendinblue = new Sendinblue("https://api.sendinblue.com/v2.0", sendinblueConfig.key)
+
+  case class SendinblueResponse(code: String, message: String, data: SendinblueResponseData)
+  case class SendinblueResponseData(`message-id`: Option[String])
+
+
+  def sendMime(
+    message: MimeMessage,
+    tags: List[String] = List.empty,
+    attachments: List[Attachment] = List.empty,
+    headers: Map[String, String] = Map.empty
+  )(
+    implicit
+    executionContext: ExecutionContext
+  ): Future[Either[MailError, MailResponse]] =
+    throw new Exception("unable to send mime messages in Sendinblue")
```

I would throw a `UnsupportedOperationException` instead